api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -342,6 +342,10 @@ message Bootstrap {
   // TODO(abeyad): Add public-facing documentation.
   // [#not-implemented-hide:]
   core.v3.TypedExtensionConfig xds_delegate_extension = 35;
+
+  // Optional XdsConfigTracer configuration, which allows tracking xDS resources and responses.
```

Comment:
Can you add some documentation explaining how this is intended to be used? Thanks.

Reply:
Thanks for taking a look! Sure, will do and it is still WIP and will make this PR more polished.